schemas/common/contact-lead.schema.json

```diff
@@ -0,0 +1,84 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/contact-lead",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "XDM Contact and Lead common B2B attributes",
+  "type": "object",
+  "description": "Common B2B attributes for contacts and leads.",
+  "definitions": {
+    "contact-lead": {
+      "properties": {
+        "xdm:type": {
+          "title": "Contact/Lead Type",
+          "description": "Type of the contact/lead.",
+          "type": "string"
+        },
+          "xdm:sourceType": {
+          "title": "Source Type",
+          "description": "Source type.",
+          "type": "string"
+        },		
+        "xdm:description": {
+          "title": "Contact/Lead Description",
+          "description": "Contact/Lead description.",
+          "type": "string"
+        },
+        "xdm:deleted": {
+          "title": "Deleted Indicator",
+          "description": "Indicator whether the instance was deleted.",
+          "type": "boolean"
+        },
+        "xdm:ownerID": {
+          "title": "Owner ID",
+          "description": "Owner id.",
+          "type": "string"
+        },
+        "xdm:emailBounced": {
+          "title": "Email Bounced Indicator",
+          "description": "If bounce management is activated and an email is sent to a contact/lead, indicates whether the email bounced (true) or not (false).",
+          "type": "boolean"
+        },
+        "xdm:emailBouncedDate": {
+          "title": "Email Bounced Date",
+          "description": "If bounce management is activated and an email is sent to a contact/lead, provides the date when the email was bounced.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "xdm:emailBouncedReason": {
+          "title": "Email Bounced Reason",
+          "description": "If bounce management is activated and an email is sent to a contact/lead, provides the reason for bounced email.",
+          "type": "string"
+        },
+        "xdm:status": {
+          "title": "Lead/Contact Status",
+          "description": "Field recording the current marketing/sales status of the lead/contact.",
+          "type": "string"
+        },
+        "xdm:timezone": {
+          "title": "Timezone",
+          "description": "Timezone of the contact's/lead's location.",
+          "type": "string"

```

Comment:
In what format?  According to what standard?

Reply:
+1 - user will want to know what format they should provide here. If we have no idea, then I would suggest removing the field until you have some sample data to refer to.

Reply:
We don't have any reference data yet so, I will remove this for now.